Makefile

```diff
@@ -143,18 +152,40 @@ clean-plugins:
 		rm -f $(BIN_PATH)/$$i; \
 	done
 
+clean-binaries:
+	$(Q)echo "Cleaning $(BINARIES)"; \
+	for i in $(BINARIES); do \
+		rm -f $(BIN_PATH)/$$i; \
+	done
+
+clean-images:
+	$(Q)echo "Cleaning exported images and deployment files."; \
+	rm -f $(IMAGE_PATH)/*
+
 clean-cache:
 	$(Q)$(GO_CMD) clean -cache -testcache
 
 #
 # plugins build targets
 #
 
+$(BIN_PATH)/nri-resource-policy-%: .static.%.$(STATIC)
+	$(Q)echo "Building $(STATIC_TYPE)$@ (version $(BUILD_VERSION), build $(BUILD_BUILDID))..."; \
+	src="./cmd/plugins/$(patsubst nri-resource-policy-%,%,$(notdir $@))"; \
+	mkdir -p $(BIN_PATH); \
+	cd "$$src" && $(GO_BUILD) $(BUILD_TAGS) $(LDFLAGS) $(GCFLAGS) -o $@
+
+$(BIN_PATH)/nri-%: .static.%.$(STATIC)
```

Comment:
What is this rule for? Should/could we rename `config-manager` to `nri-config-manager` and modify this target to build that (and possibly drop the target below)?

Reply:
No. According to the current (documented) conventions, only plugins should be named `nri-%`. Everything else is some other helper/nri-related binary.